packages/backend-api/install/tests/ShopBundle/Test/OauthTestCase.php

```diff
@@ -80,18 +82,42 @@ protected function createOauthToken(): string
 
     /**
      * @param string $method
-     * @param string $uri
-     * @param mixed|null $content
+     * @param string $path
+     * @param array|null $content
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    protected function runOauthRequest(string $method, string $uri, $content = null): Response
+    protected function runOauthRequest(string $method, string $path, ?array $content = null): Response
     {
         $client = $this->getClient();
         $headers = [
             'HTTP_AUTHORIZATION' => sprintf('Bearer %s', $this->createOauthToken()),
             'HTTP_ACCEPT' => '*/*',
+            'CONTENT_TYPE' => 'application/json',
         ];
-        $client->request($method, $uri, [], [], $headers, $content);
+        $uri = $this->getDomainBaseUrl() . $path;
+        $client->request($method, $uri, [], [], $headers, $content ? json_encode($content) : null);
```

Comment:
Well technically there is a difference. `null` sends nothing in content `[]` sends `[]` in content. When our API doesn't expect content, then it ignores it, so it doesn't matter.

But I can come with two reasons for `null`
* it's default `Client::request()` value
* We define in the documentation there should be no content

Reply:
I'm not sure we understand each other. 
I'm referring to`$content` in a condition – whether it's intentional to convert empty array in `$content` to null, because passing empty array to method `runOauthRequest` does that.

Reply:
you are right, it is incorrect. Fixed